code/script.js

```diff
@@ -1,325 +1,570 @@
+    hair: "no",
+    eyes: "blue",
+    accessories: ["armor"],
+    weapon: ["sword", "falx"],
+    type: "monster",
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hair: 'black',
-    eyes: 'brown',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Hannibal Lecter",
+    img: "images/hannibal.png",
+    hair: "brown",
+    eyes: "brown",
+    accessories: ["mask", "suit"],
+    weapon: ["mind-games"],
+    type: "human",
   },
-]
+];
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret;
+let currentQuestion;
+let charactersNotInPlay;
+let charactersInPlay;
 
 // Draw the game board
 const generateBoard = () => {
-  board.innerHTML = ''
+  console.log(charactersNotInPlay);
+  board.innerHTML = "";
   charactersInPlay.forEach((person) => {
+    console.log(charactersInPlay);
     board.innerHTML += `
-      <div class="card">
+      <div class="card border-gradient border-gradient-color kept">
         <p>${person.name}</p>
         <img src=${person.img} alt=${person.name}>
         <div class="guess">
           <span>Guess on ${person.name}?</span>
-          <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
+          <button class="guess-btn" onclick="guess('${person.name}'); playGameAudio();">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+  charactersNotInPlay.forEach((person) => {
+    console.log(charactersNotInPlay);
+    board.innerHTML += `
+      <div class="card thrown" >
+        <p>${person.name}</p>
+        
+      </div>
+    `;
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+  console.log(secret);
+};
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  charactersInPlay = CHARACTERS;
+  charactersNotInPlay = [];
+  generateBoard();
+  setSecret();
+  playGameAudio();
+  totalSeconds = 0;
+  valString = "";
+  secondsLabel.innerHTML = "";
+  minutesLabel.innerHTML = "";
+  countGuess.innerHTML = "0";
+};
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
+  const category = questions.options[questions.selectedIndex].parentNode.label;
 
   // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+  const value = questions.value;
+
+  console.log(value);
```

Comment:
Great that you are using console.log! Before your final push they can be removed